doc/sphinx/source/pocketlab/pocketlab.rst

```diff
@@ -0,0 +1,55 @@
+============
+Introduction
+============
+Pocket lab is a portable display-based embedded solution created primarily for demoing the ADCs and DACs functionality.
```

Comment:
Do we have any DAC supported in PL currently?

Reply:
No will remove this